autoexec.bat.command_com.template

```diff
@@ -0,0 +1,24 @@
+@ECHO OFF
+PROMPT $p$g
+cd c:\flash
+cls
+echo.
+echo.
+echo This will flash your EC firmware to support a classic 7-row keyboard (default)
+echo or other modifications as configured at modification time.
+echo.
+echo.
+echo WARNING: Incorrectly flashing your firmware can damage your laptop.
+echo Ensure this is the right firmware version before attempting to flash
+echo the firmware - there are minimal checks done.
+echo.
+echo This disc will flash the file __DIR\__FL2
+echo.
+echo This disk is prepared for __DESC
+echo.
+echo Buildinfo: __BUILDINFO
+echo Built: __BUILT
+echo.
+pause
+echo on
+command.com

```

Comment:
> I also wonder if the default flash scripts will allow you to re-flash a "same version" image

This seems not to be an issue here. The EC firmware could just be downgraded with the patched.x200.314.106 image/flasher. I can confirm that this worked. I also tried to increase the version numbers but this was not reflected in the BIOS message. To be continued.

> PHLASH16.EXE

This is worth a try, thanks!